bind/gencffi_func.go

```diff
@@ -32,6 +32,52 @@ def %[2]s(%[3]s):
 	g.wrapper.Printf("\n")
 }
 
+func (g *cffiGen) genGetFunc(o Func) {
+	sig := o.Signature()
+	args := sig.Params()
+
+	var funcArgs []string
+	for _, arg := range args {
+		funcArgs = append(funcArgs, arg.Name())
+	}
+	g.wrapper.Printf(`
+# pythonization of: %[1]s.%[2]s 
+def Get%[2]s(%[3]s):
+`,
```

Comment:
perhaps inject the documentation (if any) for the variable we get here?
so when we do `help(mymodule.GetFoo)` we get the documentation that was attached to the `Foo` variable on the Go side.

that would look like:

```go
g.wrapper.Printf(`
# pythonization of: %[1]s.%[2]s
def Get%[2]s(%[3]s):
    """%[4]s""""
`,
  g.pkg.pkg.Name(),
  o.GoName(),
  strings.Join(funcArgs, ", "),
  o.Doc(),
)
```